stories/topics/proc-aap-aws-deploying-extension-nodes.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc-aap-aws-deploying-extension-nodes"]
+
+.Procedure
+. To deploy the extension nodes, run the command generator to generate the CLI command.
++
+[source,bash]
+---- 
+$ docker run --rm -v $(pwd)/command_generator_data:/data $IMAGE command_generator --data-file /data/extra_vars.yml
+----
++
+Provides the following command:
```

Comment:
```suggestion
Provides the following upgrade command:
```
Just a suggestion: the next step in the procedure refers to this as the upgrade command